default/data/ui/views/rw_vc_zoom_ops.xml

```diff
@@ -17,7 +17,9 @@
       <single>
         <title>Active Meetings</title>
         <search>
-          <query>`rw_vc_indexes` sourcetype="zoom:webhook" source="meeting.started"
+          <query>`rw_vc_indexes` sourcetype="zoom:webhook" TERM(meeting.started) OR TERM(meeting.ended) event="meeting.started" OR event="webinar.started" OR event="meeting.ended" OR event="webinar.ended" 
+| stats latest(event) AS event latest(_time) AS _time BY payload.object.uuid 
+| where event="meeting.started" 
```

Comment:
Change `| where event="meeting.started"` to `| where event="meeting.started" OR event="webinar.started"` to account for webinars.